### PR TITLE
Using dedicated ServiceAccount for manager pod

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**
currently manager pod is using the default service account in order to increase readability and to avoid confusion we will use a dedicated ServiceAccount instead of default

### CODE REVIEW

File: config/manager/kustomization.yaml
Comment (marquiz) [inline config/manager/kustomization.yaml:1]: Unintended changes in this file(?)

**Review by marquiz — COMMENTED** [review]
Thanks @yevgeny-shnaidman for the PR. Good improvement, just one comment below

File: config/manager/kustomization.yaml
Comment (yevgeny-shnaidman) [reply config/manager/kustomization.yaml:1]: Thanks @marquiz for catching this one ! Fixed

**Comment by ArangoGutierrez** [comment]
And the Helm Deployment?

**Comment by yevgeny-shnaidman** [comment]
> And the Helm Deployment?

Is there a current use for Operator helm deployment?  My thought was that helm deployment should be re-visited once we unify the repos

**Review by marquiz — COMMENTED** [review]
nit: typo in the commit message (and PR title) s/dedecated/dedicated/

**Comment by yevgeny-shnaidman** [comment]
> nit: typo in the commit message (and PR title) s/dedecated/dedicated/

Fixed

**Review by marquiz — APPROVED** [review]
Looks good 👍

**Comment by ArangoGutierrez** [comment]
> > And the Helm Deployment?
> 
> Is there a current use for Operator helm deployment? My thought was that helm deployment should be re-visited once we unify the repos

forgot about that conversation
/lgtm

**Pull request merged**